tests/product/test_server_install.py

```diff
@@ -227,24 +227,6 @@ def test_install(self, installer=None):
             self.assert_has_default_config(container)
             self.assert_has_default_connector(container)
 
-    def test_install_worker_is_pa_master(self):
```

Comment:
I think we could also remove test_install because test_install_with_java8_home covers that (and all of our customers set java8_home b/c it's in the docs and they copy paste). test_install_with_java8_home can be in the smoketest